Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -34,12 +34,10 @@
 NSString* const UIPasteboardChangedTypesAddedKey = @"UIPasteboardChangedTypesAddedKey";
 NSString* const UIPasteboardChangedTypesRemovedKey = @"UIPasteboardChangedTypesRemovedKey";
 
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListString = StubConstant();
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListURL = StubConstant();
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListImage = StubConstant();
+NSArray<NSString*>* const UIPasteboardTypeListString = @[ (NSString*)kUTTypeUTF8PlainText, (NSString*)kUTTypeText ];
+NSArray<NSString*>* const UIPasteboardTypeListURL = @[ (NSString*)kUTTypeURL ];
+NSArray<NSString*>* const UIPasteboardTypeListImage = @[ (NSString*)kUTTypePNG, (NSString*)kUTTypeJPEG ];
+
```

Comment:
Thx! we discussed here, use 3 and added comment to make it obvious

---
In reply to: [87642571](https://github.com/Microsoft/WinObjC/pull/1358#discussion_r87642571) [](ancestors = 87642571,87642191,87641964)